docs/running_rune_with_occlum_bundle.md

```diff
@@ -48,14 +48,14 @@ and then configure enclave runtime as following:
 ``` json
   "annotations": {
       "enclave.type": "intelSgx",
-      "enclave.runtime.path": "/usr/lib/liberpal-occlum.so",
+      "enclave.runtime.path": "/opt/occlum/build/lib/libocclum-pal.so.0.14.0",
       "enclave.runtime.args": "$OCCLUM_INSTANCE_DIR"
   }
 ```
 
 where:
 - @enclave.type: specify the type of enclave hardware to use, such as `intelSgx`.
-- @enclave.runtime.path: specify the path to enclave runtime to launch. For an Occlum application, you need to specify the path of `liberpal-occlum.so` which is a soft link to `libocclum-pal.so` of your Occlum instance dir (`OCCLUM_INSTANCE_DIR`).
+- @enclave.runtime.path: specify the path to enclave runtime to launch. For an Occlum application, you need to specify the path of `libocclum-pal.so.0.14.0`.
```

Comment:
I think the version number is not necessary here. Because you are controlling the package, so you can control where the pal is installed. So occlum sdk forgets to create a symbol link to libocclum-pal.so.0.14.0, but you can add the symbol link in your package, e.g, install libocclum-pal.so.0.14.0 to /usr/lib/libocclum-pal.so.0.14.0 and create a symbol link /usr/lib/libocclum-pal.so, right?